Feature/cjm 62503 move the xdm:trackingToken from option-detail.schema.json to decisionitem.schema.json

kenadobe opened the pull request:
#1828

usankara reviewed (approved):
LGTM!

usankara commented on extensions/adobe/experience/decisioning/decisionitem.schema.json:
```diff
@@ -69,6 +69,11 @@
           "title": "Item Calendar Constraints",
           "description": "Defines the date range during which this decision item is eligible. This constraint may not be strictly enforced always."
+        },
+        "xdm:_trackingToken": {
```
missed to notice the path -  let's double check this.

kenadobe replied:
The team decided to change the path to be consistent with all other attributes. I have updated it to "https://ns.adobe.com/experience/decisioning/decisionitem/trackingToken"

kenadobe commented:
I updated the trackingToken path to be consistent with all other attributes in the schema and also removed the "_" prefix from the trackingToken attribute name.

anandphatak commented on extensions/adobe/experience/decisioning/option-detail.schema.json:
```diff
@@ -104,6 +104,7 @@
           "description": "Option selection type. The selection type of which the item was derived from."
         },
         "xdm:trackingToken": {
+          "meta:status": "deprecated",
```
@kenadobe , just FYI, marking a field `deprecated` merely means that it will not show up in the UI. But someone can still ingest data in this field.
